File: Program Cpp/Car.cpp
#pragma once

#include <iostream>
#include <string>
#include <vector>
#include "Vehicle.cpp"

using namespace std;

class Car : public Vehicle
{
	// Class Car melakukan Inheritance kepada kelas Vechicle 

private:
	string jumlah_kursi;
	string jumlah_pintu;
	// Variabel yang dibutuhkan
	
public:
	Car(string plat_nomer, string merk, string tahun_produksi, string warna, string jumlah_pintu, string jumlah_kursi) : Vehicle(plat_nomer, merk, tahun_produksi, warna){
		this->jumlah_pintu = jumlah_pintu;
		this->jumlah_kursi = jumlah_kursi;
	}
	// Constructor

	string getJumlahPintu(){
		return this->jumlah_pintu;
	}
	
	string getJumlahKursi(){
		return this->jumlah_kursi;
	}

	string getDescription(){
		return "Jumlah Pintu: " + this->jumlah_pintu + ", " + "Jumlah Kursi: " + this->jumlah_kursi;
	}
};

File: Program Python/Car.py
from Vehicle import Vehicle

class Car(Vehicle):
    def __init__(self, plat_nomer, merk, tahun_produksi, warna, jumlah_pintu, jumlah_kursi):
        super().__init__(plat_nomer, merk, tahun_produksi, warna)
        self.jumlah_pintu = jumlah_pintu
        self.jumlah_kursi = jumlah_kursi

    def get_jumlah_pintu(self):
        return self.jumlah_pintu

    def get_jumlah_kursi(self):
        return self.jumlah_kursi

    def get_description(self):
        return f"Jumlah Pintu: {self.jumlah_pintu}, Jumlah Kursi: {self.jumlah_kursi}"